lib/mainscreen.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'Seconscreen.dart';
+
+class MainScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("My App"),
+        centerTitle: true,
+        backgroundColor: Colors.orangeAccent,
+        automaticallyImplyLeading: false,
+      ),
+      body: Container(
+        alignment: Alignment.center,
+        padding: EdgeInsets.only(top: 16.0),
+        margin: EdgeInsets.all(100.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: <Widget>[
+            SizedBox(
+              width: 400.0,
```

Comment:
If you keep the margin 100 from left and right, you will never get a box of width 400.  Instead of `SizedBox` use `Expanded`.

For example

```dart
Expanded(
  child: Card(
   .. 
  ) 
)
```

Also between three expanded, have a SizedBox of height 16